tdrs-backend/manifest.yml

```diff
@@ -1,8 +1,20 @@
 version: 1
 applications:
 - name: tdp-backend
-  memory: 256M
+  memory: 512M
```

Comment:
Our previous memory limit seems to impair the backend application performance: 
![Screenshot from 2021-04-20 14-40-35](https://user-images.githubusercontent.com/22626085/115465916-4ddc1980-a1fd-11eb-8359-fbd8019af0e5.png)